frontend_server/lib/server.dart

```diff
@@ -227,6 +255,7 @@ class _FrontendCompiler implements CompilerInterface {
     return Uri.base.resolve(path);
   }
 
+
```

Comment:
nit: here and below new blank lines seem unintended

Reply:
removed. cleaned up